### ISSUE TITLE
minor nit: install instructions cause error

### ISSUE THREAD

**Issue by ultrasaurus** [openopps/openopps-platform#1115]
in https://github.com/18F/midas/blob/dev/INSTALL.md, it says:

```
nvm install 0.10
nvm alias default 0.10
nvm version             # should be something like v0.10.38
```

but when I do that I get:

```
$ nvm install 0.10
######################################################################## 100.0%
Now using node v0.10.40 (npm v1.4.28)
$ nvm alias default 0.10
! WARNING: Version '0.10' does not exist.
default -> 0.10 (-> N/A)
```

maybe it should be

```
nvm install 0.10
nvm version             # should be something like v0.10.38
nvm alias default 0.10.38    # or whatever version you have installed
```

**Comment by dhcole** [comment]
Hmm, yea, that would work, or, we could pin installation to a patch version like `nvm install 0.10.40` `nvm alias default 0.10.40`. @jeremiak any thoughts on the best way to do this? I think you're more of an `nvm` user than me.

**Comment by jeremiak** [comment]
I think pinning would be the best. @ultrasaurus just from looking at your terminal output above, I think you'll also need to update `npm` since we're using the `2.1.x` range and I guess node `0.10.40` ships with `npm v1.4.28`. We should update the docs to reflect that as well. `nvm` has a shim for `npm` so once you `nvm use midas` you should also run `npm install -g npm@2.1.x`

**Comment by rogeruiz** [comment]
According to the NPM documentation [here](https://docs.npmjs.com/getting-started/installing-node#updating-npm), `sudo` should be used to upgrade `npm`.

I also ran into some issues where if you install `node` using `brew install node` using a node manager, I specifically was using `n` and not `nvm`. The issue I ran into was a permissions issue in the `man` directory being symlinked to the `brew` `/usr/local/Cellar/` install of `node` and the upgrade for `npm` kept failing–surprisingly even when using `sudo`.

I can better explain the issue, but I think it might be a good idea to figure out a way to install the version of `node` and `npm` needed specifically for the project while mentioning that there are some caveats when upgrading `npm` for the first time because of `0.10.40`'s `npm` install being so out of date with version needed in the `package.json`.
